modules/administration-guide/partials/proc_restoring-a-che-instance-from-a-backup-by-snapshot-id.adoc

```diff
@@ -0,0 +1,50 @@
+[id="restoring-a-{prod-id-short}-instance-from-a-backup-by-snapshot-id_{context}"]
+= Restoring a {prod-short} instance from a backup by snapshot ID
+
+To restore a {prod-short} instance from a particular backup snapshot, you can enter the snapshot ID with `{prod-cli}`.
```

Comment:
It's just the intro sentence, I'd like to avoid making it read like it is the procedure. Also, "can" implies a possibility, not the (only) way to do so, but one possible way.